### ISSUE TITLE
Gem install hangs - fatal: Not a git repository

### ISSUE THREAD

**Issue by jasonivers** [danielpclark/faster_path#90]
I just updated from Ruby 2.3.1 to 2.3.3, tried to bundle install, and found it kept hanging on faster_path, so I tried to install it individually, and got this:

gem install faster_path --version 0.1.8 --verbose

```
...
checking for Dummy Makefile... no
checking for cargo... yes
fatal: Not a git repository (or any of the parent directories): .git
```

**Comment by danielpclark** [comment]
Sounds like the gemspec needs to be modified to not depend on git.

**Comment by danielpclark** [comment]
I've updated master branch so that the `gemspec` no longer uses git to list files.  It's not release in gem form but perhaps this will amend the issue you had.

**Comment by jasonivers** [comment]
Still hanging for me, but I haven't had time to figure out how to make bundle be verbose about the gem install, rather than just its own verbose thing, so I don't know if it's the same issue or something else.

**Comment by danielpclark** [comment]
This should be fixed with the 0.1.10 gem release so I'm closing this.
